pkg/capture/constants/create_capture_defaults.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package constants
+
+import "time"
+
+// Default values for capture create command
+const (
+	DefaultBlobUpload         string        = ""
+	DefaultDebug              bool          = false
+	DefaultDuration           time.Duration = 1 * time.Minute
+	DefaultExcludeFilter      string        = ""
+	DefaultHostPath           string        = "/mnt/retina/captures"
+	DefaultIncludeFilter      string        = ""
+	DefaultIncludeMetadata    bool          = true
+	DefaultJobNumLimit        int           = 0
+	DefaultMaxSize            int           = 100
+	DefaultNamespaceSelectors string        = ""
+	DefaultNodeNames          string        = ""
+	DefaultNodeSelectors      string        = "kubernetes.io/os=linux"
+	DefaultNowait             bool          = true
+	DefaultPacketSize         int           = 0
+	DefaultPodSelectors       string        = ""
+	DefaultPVC                string        = ""
+	DefaultS3AccessKeyID      string        = ""
+	DefaultS3Bucket           string        = ""
+	DefaultS3Endpoint         string        = ""
+	DefaultS3Path             string        = "retina/captures"
+	DefaultS3Region           string        = ""
+	DefaultS3SecretAccessKey  string        = ""
+	DefaultTcpdumpFilter      string        = ""
+	DefaultWaitPeriod         time.Duration = 1 * time.Minute
+	DefaultWaitTimeout        time.Duration = 5 * time.Minute
```

Comment:
I don't think these do much to improve readability... they just add a layer of indirection when navigating the flags. If a linter demands this, I'd say just silence it.

Reply:
I don't mind it IFF they're in the commands' package instead of centralized like this where you have to hunt for them

Reply:
Following from the comment in command package, we can possible have have a single const for go defaults i.e. empty string, false bool and 0 int

Reply:
Thanks for feedback, got rid of the `create_capture_defaults` file and moved the defaults to the create commands package directly. Also got rid of all the empty defaults as suggested. Hope this is a happy compromise.